### PR TITLE
fix replaces binding

### PR DESCRIPTION
**Pull request by jneilliii**

### CODE REVIEW

File: octoprint_virtual_printer_settings/__init__.py [hunk 1]
```diff
@@ -11,26 +11,29 @@
...
+			type="settings",
+			name="Virtual Printer",
+			replaces="plugin_virtual_printer",
+			template="virtual_printer2_settings.jinja2",
 			custom_bindings=False
 		)]
 
+	def get_assets(self):
+		return dict(
+			js=["js/jquery-ui.min.js", "js/knockout-sortable.js", "js/virtual_printer_settings.js"]
```
Comment (cp2004) [inline octoprint_virtual_printer_settings/__init__.py:28]: Are these part of developing the list views? Doesn't looked like they've been added to the PR

**Review by cp2004 — APPROVED** [review]
All seems good, apart from the extra JS files. Although OctoPrint doesn't seem to care that they're not there, as it works.

**Comment by cp2004** [comment]
Thanks for fixing my copy/paste errors too, I thought I'd got all of them...

File: octoprint_virtual_printer_settings/__init__.py (hunk 1 above)
Comment (jneilliii) [reply octoprint_virtual_printer_settings/__init__.py:28]: oh yeah, did I not push those...let me fix that for you.

**Comment by jneilliii** [comment]
no problem, I can't live well enough alone when I see something.

**Comment by jneilliii** [comment]
I just pushed those two files, should be good now.

**Comment by cp2004** [comment]
Cool, will get to work learning how to represent the lists. Thanks!

**Comment by cp2004** [comment]
I hit the wrong button....

**Pull request merged**